### DISCUSSION TITLE
Design Pattern in Learning Resources Guide

### DISCUSSION THREAD

**Discussion by Niravanaa** [NathanGrenier/SOEN-341#9] [Ideas]
Maybe a few resources for the MVC design pattern can be added in the resource guide page. It would greatly help us as we design our web application

https://www.freecodecamp.org/news/model-view-controller-mvc-explained-through-ordering-drinks-at-the-bar-efcba6255053/
https://www.freecodecamp.org/news/the-model-view-controller-pattern-mvc-architecture-and-frameworks-explained/
https://www.tutorialspoint.com/mvc_framework/mvc_framework_introduction.htm

**Comment by NathanGrenier** [comment]
Not sure if svelte follows that paradigm. For the most part, how we work with data is handled by the frameworks we use. Still an interesting read though.
https://stackoverflow.com/questions/58515541/does-svelte-do-away-with-mvc
